cloud/create-databases.yml

```diff
@@ -0,0 +1,19 @@
+- hosts:
+    - concent-builder
+  vars_files:
+    - ../concent-builder/consts.yml
+    - "{{ deployment_values }}/var.yml"
+    - "{{ deployment_values }}/var-{{ cluster }}.yml"
+    - "{{ local_secret_dir }}/{{ cluster }}/cluster-secrets.yml"
+    - "{{ local_secret_dir }}/cloud/cloud-secrets.yml"
+  vars:
+    choose_database_option: create
+  roles:
+    - {
+        role: postgres_connection,
+        cluster_type: control,
```

Comment:
I'd make it more compact:

``` yaml
    - {role: postgres_connection, cluster_type: control}
    - {role: postgres_connection, cluster_type: storage}
```

Reply:
@cameel I did that before but you told me to fix my style.

Reply:
See https://github.com/golemfactory/concent-deployment/pull/226#discussion_r218452341

Reply:
Done